Master-Game/World/Map.cs

```diff
@@ -39,5 +53,91 @@ public void ResetTiles()
                 }
             }
         }
+
+        public MapData LoadJson()
+        {
+            string fileName = "Jay01.json";
+            string filePath = Path.GetDirectoryName(Path.GetDirectoryName(System.IO.Directory.GetCurrentDirectory())) + "\\Assets\\Maps\\" + fileName;
+            FileInfo f = new FileInfo(fileName);
+            string fullName = f.Name;
+            using (StreamReader reader = new StreamReader(filePath))
+            {
+                string myJson = reader.ReadToEnd();
+                MapData mapData = JsonConvert.DeserializeObject<MapData>(myJson);
+                return mapData;
+            }
+        }
+
+        public class MapData
+        {
+            //Name
+            public string name { get; set; }
+            //Version
+            public string version { get; set; }
+            //row
+            public int rows { get; set; }
+            //col
+            public int cols { get; set; }
+            //Tiles
+            public Tiles [] tiles { get; set; }
+            //Entities
+            public Entities[] entities { get; set; }
+
+            public int GetRows()
+            {
+                return rows;
+            }
+        }
+
+        public class Tiles
```

Comment:
I saw on a message board that this was a good way to handle JSON arrays. Let me know if you can think of a better way to handle arrays.

Reply:
This seems fine